packages/core/bridge/helpers.ts

```diff
@@ -0,0 +1,37 @@
+const isVersionedTransaction = (tx: any) => {
+    return "version" in tx;
+};
+
+/**
+ * @solana/web3.js only exports Node Wallet
+ * but we need to export a wallet that can be used in the browser
```

Comment:
I'm curious how this approach compares to https://github.com/anza-xyz/wallet-adapter

This is one of the main wallet libraries used in Solana. A lot of the guts are here: https://github.com/anza-xyz/wallet-adapter/blob/master/packages/core/base/src/adapter.ts

Reply:
How can we achieve the organization they do of different directories per wallet https://github.com/anza-xyz/wallet-adapter/tree/master/packages/wallets